### ISSUE TITLE
[BUG] Dragging over tokens causes "infinity" distance and red warning

### ISSUE THREAD

**Issue by ctbritt** [ironmonk108/enhanced-terrain-layer#131]
**Describe the bug**
Dragging a token over another token with "Include Live/Dead Tokens" selected in settings results in a red warning square and an "infinity" distance. 

**To Reproduce**
1. Drag a token over another token
2. Watch the red warnings appear.

**Expected behavior**
Token should count the intervening token as difficult terrain and double the movement cost.

**Screenshots**
![CleanShot 2023-03-02 at 17 27 11](https://user-images.githubusercontent.com/4516034/222574951-e160e119-820a-490d-b8ce-e36e905d8c50.gif)

**Please complete as much of the following information as possible:**
 - Browser Version 110.0.5481.177 (Official Build) (arm64)
 - Foundry Version 10.291
 - System DnD 2.1.5
 - Module version 10.6
 - Are there any errors in the console No
 - Have you tried using `Find the Culprit` to make sure it isn't a module interaction. Yes. Does not seem to be module interaction. 

**Additional context**

**Comment by ctbritt** [comment]
Addendum. Seems to be a conflict with [DnD5e Drag Ruler Integration](https://github.com/PepijnMC/ElevationDragRuler)

**Comment by jputman** [comment]
Agreed that I have been seeing this issue, and can confirm that disabling the Token Difficult Terrain check in DnD5e Drag Ruler Integration resolves the issue. But it seems to only impact hostile creatures. Neutral and Friendly seem to calculate properly.

**Comment by davelacorneille** [comment]
Same for me.

**Comment by ctbritt** [comment]
Have fixed this. Set the `Rule Provider` setting in Enhanced Terrain Layer to `Built-In.` Also set `Include Live/Dead Tokens` settings to `Any`... Then, in the D&D Drag Ruler Integration setting, set `Token Difficult Terrain` to true. This fixed the "infinity" issue for me.

**Comment by ironmonk88** [comment]
So, this isn't an Enhanced Terrain Issue.  And... I'm not actually sure it's an issue per say.
The Rule Provider from the ruler is returning Infinity because it's saying, due to rules, you can't walk through that square.
Drag Ruler isn't showing this information very well.

So if you think the token should be able to walk through the square, and the ruler is saying you can't, then it's an issue with the ruler module.
But also, Drag Ruler should have a ticket opened to maybe change "Infinity" to "&infin;" for better readability.

**Comment by jputman** [comment]
I think the mechanics issue is that the token is marked dead and thus should only be difficult terrain and not blocking terrain.
